### DISCUSSION TITLE
Custom style for table

### DISCUSSION THREAD

**Discussion by Starina123** [requarks/wiki#3777] [Error / Bug Report]
Example

```markdown
| col1 |  col2 |
| - |  - |
| val1 |  val2 |{.mystyle}
````

When I write such a construction, the following code is generated in the editor 

```html
<table class="mystyle">
```

When the code for the view is generated, this style is absent. 

```html
<table>
```

**Comment by srob411** [comment]
Same problem here (wiki.js 2.5.191). It is rendered in the editor preview correctly, but in the newly rendered page the class is gone. It is working for headings and paragraphs, though...

How do you know that it is supposed to work? I cannot find it documented anywhere.

**Reply by Starina123** [reply]
The documentation is incomplete. 
I researched the sources of the [wiki-docs](https://github.com/Requarks/wiki-docs). Similar syntax is used in many places (e.g. {.grid-list}, {.caption}, {.links-list}). I did it by analogy.

If this feature works for many elements, it would be great.

**Comment by Starina123** [comment]
Correct spelling
```
| col1 |  col2 |
| - |  - |
| val1 |  val2 |
{.mystyle}
```

The style must be specified on a new line
